src/audit-allowlist.json

```diff
@@ -19,6 +19,14 @@
         {
             "id": "CVE-2023-45133",
             "reason": "indirect dependency of a few babel packages. should be looked at as a seperate ticket. see: https://trello.com/c/alXeUZpI/1918-fix-dependency-issues-with-babel-traverse"
+        },
+        {
+            "id": "CVE-2024-4067",
+            "reason": "Micromatch, dependency of jest. should be looked at as a seperate ticket. context: https://github.com/micromatch/micromatch/issues/243"
+        },
+        {
+            "id": "CVE-2024-4068",
+            "reason": "Braces, dependency of micromatch, indirect dependency of jest. context: https://github.com/micromatch/micromatch/issues/243"
```

Comment:
Have you created tickets for this?

Reply:
No m, will do that now. Thank you Linden.

Reply:
Braces is resolved with v3.0.3